connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/WorkerHandle.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.connect.util.clusters;
+
+import org.apache.kafka.connect.cli.ConnectDistributed;
+import org.apache.kafka.connect.runtime.Connect;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * A handle to a worker executing in a Connect cluster.
+ */
+public class WorkerHandle {
+    private static final Logger log = LoggerFactory.getLogger(WorkerHandle.class);
+
+    private final String workerName;
+    private final Connect worker;
+
+    protected WorkerHandle(String workerName, Connect worker) {
+        this.workerName = workerName;
+        this.worker = worker;
+    }
+
+    /**
+     * Create and start a new worker with the given properties.
+     *
+     * @param name a name for this worker
+     * @param workerProperties the worker properties
+     * @return the worker's handle
+     */
+    public static WorkerHandle start(String name, Map<String, String> workerProperties) {
+        return new WorkerHandle(name, new ConnectDistributed().startConnect(workerProperties));
+    }
+
+    /**
+     * Stop this worker.
+     */
+    public void stop() {
+    }
```

Comment:
Does this method intentionally do nothing?

Reply:
Funny right? I noticed yesterday too. Thanks! 
I removed the `process()` that gives access to the underlying `Connect` field and added facades for the rest now, including `url` and `stop`